webclient/elements

```diff
@@ -0,0 +1 @@
+../elements
```

Comment:
`webclient/elements` symlinks to the root directory, but `webclient/data` symlinks to the common directory. Why the inconsistency?